content/TruenasApps/StableApps/InstallNextCloudMedia.md

```diff
@@ -15,151 +15,276 @@ keywords:
-   This example uses the default *nextcloud*.
+{{< expand "Nextcloud Cron Jobs" "v" >}}
+NextCloud cron jobs only run while the app is running. If you stop the app, the cron job(s) do not run until you start the app again.
 
-3. Scroll down to or click on **Nextcloud Configuration** to show the app configuration settings.
-   For a basic installation you can leave the default values in all settings except **Username** and **Password**.
+For more information on formatting and using cron jobs, see [Managing Cron Jobs]({{< relref "ManageCronJobsSCALE.md" >}}).
+{{< /expand >}}
 
-   a. Enter the username and password created in the [Before You Begin](#before-you-begin) section or for the existing Nextcloud administrator user account credentials.
-      This example uses *admin* as the user.
+The TrueNAS app is configured with all the required environment variables, but if you want to further customize the container, click **Add** to the right of **Additional Environment Variables** for each to enter the variable(s) and values(s).
+{{< /expand >}}
+Enter the network configuration settings. 
+Enter the default port **30027** in **WebUI Port**.
 
-   {{< trueimage src="/images/SCALE/Apps/InstallNextcloudConfigUsernamePassword.png" alt="Nextcloud Configuration Username" id="Nextcloud Configuration Username" >}}
+If you configured a certificate, select it in **Certificate ID**. A certificate is required if you want to select a different external port.
 
-   TrueNAS populates **Host** with the IP address for your TrueNAS server and **Nextcloud data directory** populates with the correct path.
+Enter the storage settings for each of the datasets created for the Nextcloud app.
+{{< expand "Configuring Nextcloud Storage" "v" >}}
+Do not select **DEPRECATED: Old Storage Structure** if you are deploying Nextcloud for the first time as this slows down the installation and is not necessary.
+If you are upgrading where your Nextcloud deployment in TrueNAS was a 1.x.x release, select this option.
+  
+Set **Host Path (Path that already exists on the system)** in **Type** for **Nextcloud AppData Storage**.
+Select **Enable ACL**, and then either enter or browse to and select the **html** dataset to populate the **Host Path** field.
 
-   b. Click **Add** to the right of **Command** to show the **Command** field then click in that field and select **Install ffmpeg** to automatically install the FFmpeg utility when the container starts.
+{{< trueimage src="/images/SCALE/Apps/InstallNextcloudStorageAppDataACLandACESettings.png" alt="Add Nextcloud Storage for AppData" id="Add Nextcloud Storage for AppData" >}}
 
-   {{< trueimage src="/images/SCALE/Apps/InstallNextcloudConfigAddInstallffmpetCommand.png" alt="Add Install ffmpeg Command" id="Add Install ffmpeg Command" >}}
+Select **Add** to the right of **ACL Entries**, add the **1001** user, and give it **FULL_CONTROL Access**.
```

Comment:
Image shows **Force Flag** selected and it is not mentioned in the text. Same for the next dataset.